src/servicebus/azext_servicebus/_help.py

```diff
@@ -0,0 +1,497 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['servicebus'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus namespace, queue, topic, subscription, rule and geo disaster recovery configuration - alias
+
+    """
+
+helps['servicebus namespace'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus namespace and authorization-rule
+
+    """
+
+helps['servicebus queue'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus queue and authorization-rule
+
+    """
+
+helps['servicebus topic'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus topic and authorization-rule
+
+    """
+
+helps['servicebus subscription'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus Subscription
+
+    """
+
+helps['servicebus rule'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus rule
+
+    """
+
+helps['servicebus georecovery-alias'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus Geo Disaster Recovery Configuration - Alias
+
+    """
+
+helps['servicebus namespace check-name-availability'] = """
+    type: command
+    short-summary: check for the availability of the given name for the Namespace
+    examples:
+        - name: Create a new topic.
```

Comment:
The description title is wrong?
How does `servicebus namespace check-name-availability` create a new topic?

Reply:
Resolved